### PR TITLE
Move some variables and logic surrounding the values of variables to hiera yaml files

### PR DESCRIPTION
**Pull request by bschonec**
I think it makes sense to move all of these if checks out to Hiera structured data.

### CODE REVIEW

**Comment by bschonec** [comment]
I apologize sincerely for all of the noise trying to submit these changes.  I'm no Puppet developer and Travis is something I have yet to understand.

File: manifests/server.pp [hunk 1]
```diff
@@ -50,18 +33,23 @@
...
-        notify => Service[$servicename],
+
+    # Manage the installation of the rsyncd package?
+    if $manage_package {
+
+      # RHEL8 and newer (and their variants) have a separate package for rsyncd daemon.  If the $package_name
+      # variable is defined (the variable is defined in the hiera hierarchy), then install the package.
+      if $package_name {
+        package {$package_name:
+          ensure => $rsync::package_ensure,
```
Comment (kajinamit) [inline manifests/server.pp:44]: do we also want to expose this as a parameter ?

**Review by kajinamit — COMMENTED** [review]
Thanks. This looks nice. Let me leave Just one suggestion.

**Comment by bschonec** [comment]
Yes, I'd do that but I suggest exposing the parameter in another pull request to keep the changes small.

**Review by kajinamit — APPROVED** [review]

**Pull request merged**